src/MiniCover/Reports/XmlReport.cs

```diff
@@ -71,16 +65,6 @@ public static void Execute(InstrumentationResult result, string output, float th
                                 new XAttribute(XName.Get("excluded"), "false"),
                                 new XAttribute(XName.Get("document"), Path.Combine(result.SourcePath, file.Key))
                                 );
-                            var methodHits = hits.ForMethod(methodPoint.Id).ToArray();
-
-                            var tests = methodHits.OfType<Hit.HitWithTestsInformation>().Select(h => new XElement(XName.Get("tests"),
```

Comment:
Removed this part from XmlReport, as I mentioned some days ago, I'm afraid leaving it here affects tools that reads this xml, because this XML is based on NCover XML.

Also, I don't think any tool will use that test information, I would prefer to see it on our HTML report first.